### PR TITLE
Update CHANGELOG for 5.10.0 Release

### PR DESCRIPTION
**Pull request by wesleycyu**
## What is this change?
This change updates the `CHANGELOG` for the `5.10.0` Release

## Why is this change necessary?
To keep a record of changes going into the `5.10.0` Release

## Does your change need a Changelog entry?
Yep.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +20,7 @@ populate, and send them to the Tessen service.
 - Added `sensuctl cluster id` which exposes the unique sensu cluster id.
 
 ### Changed
-- [Web] Updated embedded web assets from `275386a` ... `b0c1138`
+- [Web] Updated embedded web assets from `275386a` ... `46cd0ee`
```
Comment (jamesdphillips) [inline CHANGELOG.md]: Is this true? The ref still says `b0c1138`

Comment (wesleycyu) [reply CHANGELOG.md]: Ah, cool  -- I may have done this wrong? I ran `go generate ./` in the `release/5.10.0` branch.

Comment (jamesdphillips) [reply CHANGELOG.md]: yeah, my bad, I was looking at the master branch.

**Comment by wesleycyu** [comment]
Whooopies -- thanks @ccressent! Fixed!

**Pull request merged**